rule/cast_expr.go

```diff
@@ -51,3 +56,52 @@ func (n *exprIntToFloat) Eval(params Params) (*Value, error) {
 	}
 	return Float64Value(float64(iVal)), nil
 }
+
+/////////////////////////
+// FloatToInt Operator //
+/////////////////////////
+
+// exprFloatToInt is an operation that converts a Float64 type to an Int64 type.
+type exprFloatToInt struct {
+	operator
+}
+
+// newExprFloatToInt returns a pointer to a exprFloatToInt operator, initialised with a Contract.
+func newExprFloatToInt() *exprFloatToInt {
+	return &exprFloatToInt{
+		operator: operator{
+			contract: Contract{
+				OpCode:     "floatToInt",
+				ReturnType: INTEGER,
+				Terms: []Term{
+					{
+						Cardinality: ONE,
+						Type:        FLOAT,
+					},
+				},
+			},
+		},
+	}
+}
+
+// FloatToInt is an operator that converts as Float64 into an Int64 type.
+func FloatToInt(vN ...Expr) Expr {
```

Comment:
I am wondering why are you passing a list of Expr whereas the cardinality is one?

Reply:
Ah.. quite right, I'll fix that.

Reply:
The same goes for the `IntToFloat` function.

Reply:
Done.